back/spec/acceptance/users_spec.rb

```diff
@@ -329,7 +329,27 @@
           expect(json_response[:data].size).to eq 6
         end
 
-        example_request 'List all users includes user blocking related data' do
+        example 'List all users does not include inactive users', document: false do
```

Comment:
I still think it's enough to have only 1-2 acceptance tests on a feature. And all other cases can be covered by model unit tests like here https://github.com/CitizenLabDotCo/citizenlab/pull/4203/files#diff-248e6600d9e8f3b06c088adf0f6aead2ccea0cde3078e7b2789a4b1e4e8a2e10R493

But up to you.